community/zk-compression-airdrop/src/app/page.tsx

```diff
@@ -0,0 +1,36 @@
+'use client'
```

Comment:
nit pick: we could push the account to a leaf component and avoid making the full age client.